docker-compose.yml

```diff
@@ -60,31 +60,33 @@ services:
         condition: service_healthy
     env_file: .env
     environment:
-      - RF_LOG_LEVEL=INFO
-      - TILE_SERVER_LOCATION=http://localhost:8080
+      - RF_LOG_LEVEL=DEBUG
+      - TILE_SERVER_LOCATION=
       - COURSIER_CACHE=/root/.coursier
     ports:
       - "9000:9000"
       - "9010:9010"
     volumes:
       - ./app-backend/:/opt/raster-foundry/app-backend/
       - ./scratch/:/opt/raster-foundry/scratch/
+      - ./data/:/opt/data/
+      - $HOME/.sbt:/root/.sbt
       - $HOME/.coursier:/root/.coursier
-      - ./.sbt:/root/.sbt
       - ./.bintray:/root/.bintray
-      - $HOME/.aws:/root/.aws:ro
       - $HOME/.ivy2:/root/.ivy2
-    working_dir: /opt/raster-foundry/app-backend/
-    entrypoint: ./sbt
+      - $HOME/.aws:/root/.aws:ro
+    working_dir: /opt/raster-foundry/app-backend/api/target/scala-2.11/
```

Comment:
We should confirm that JARs aren't being built for 2.12 too. I saw some comments about 2.12 in Slack, but not sure we merged them in yet.

Reply:
It looks like there are only `2.11` jars.

Reply:
That's correct - we had an issue to upgrade if it was easy, but that didn't turn out to be the case so we deferred taking additional steps